### PR TITLE
Update download.md

### PR DESCRIPTION
**Pull request by endbacher**
re-order of menu points, link to provider database.

### CODE REVIEW

File: en/download.md [hunk 1]
```diff
@@ -4,17 +4,18 @@ lang: en
...
-# Links
-
-* [FAQ multiclient](help#multiclient) page on how to synchronize Desktop with another Delta app. 
-* [Verify Downloads](verify-downloads)
-
-## Preview Builds:
-* [Desktop](https://download.delta.chat/desktop/preview/)
-
-## Changelogs
+# Changelogs
```
Comment (ghost) [inline en/download.md]: Thanks for the initiative! Can you quickly turn all headlines into h2?

File: en/download.md [hunk 2]
```diff
@@ -4,17 +4,18 @@ lang: en
...
 * [Desktop](https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md)
 * [Android]({% include changelog-url %})
 * [iOS](https://github.com/deltachat/deltachat-ios/blob/master/CHANGELOG.md)
 * [Core](https://github.com/deltachat/deltachat-core-rust/blob/master/CHANGELOG.md)
+
+# Links
+
+* [Verify Downloads](verify-downloads): Verify data integrity of downloads
+* [Provider Database](https://providers.delta.chat/): Does my Provider work with Delta Chat?
+* [FAQ multiclient](help#multiclient): How to synchronize Desktop with another Delta Chat app.
```
Comment (ghost) [inline en/download.md]: I suggest to move verify downloads to the bottom of these 3 points; while it is important for security, 99% of our users will want to know the other two things first.

**Review by ghost — CHANGES_REQUESTED** [review]

File: en/download.md (hunk 1 above)
Comment (comradekingu) [reply en/download.md]: Done

File: en/download.md (hunk 2 above)
Comment (comradekingu) [reply en/download.md]: Done

**Pull request merged**